src/core/song.cpp

```diff
@@ -862,26 +868,26 @@ void Song::ToProtobuf(spb::tagreader::SongMetadata *pb) const {
   pb->set_performer(DataCommaSizeFromQString(d->performer_));
   pb->set_grouping(DataCommaSizeFromQString(d->grouping_));
   pb->set_lyrics(DataCommaSizeFromQString(d->lyrics_));
-  pb->set_track(d->track_);
-  pb->set_disc(d->disc_);
-  pb->set_year(d->year_);
-  pb->set_originalyear(d->originalyear_);
+  if(d->track_) pb->set_track(d->track_.value());
```

Comment:
Please follow the code-style here and all similar lines, space between if and parentheses.
`if (d->track_) pb->set_track(d->track_.value());`